Quartex.TableStorage.Benchmark/Benchmarks/GetDocumentsBenchmark.cs

```diff
@@ -0,0 +1,93 @@
+﻿using BenchmarkDotNet.Attributes;
+using Quartex.TableStorage.Benchmark.Storages;
+
+namespace Quartex.TableStorage.Benchmark.Benchmarks;
+[MemoryDiagnoser]
+public class GetDocumentsBenchmark
+{
+    private readonly BenchmarkSetup _setup = new(nameof(GetDocumentsBenchmark));
+
+    [GlobalSetup]
+    public void Setup()
+    {
+        _setup.SetupAsync().GetAwaiter().GetResult();
+        var options = new SetupOptions
+        {
+            PageSizes = [10, 20, 50, 100]
+        };
+        _setup.InsertDocuments(options).GetAwaiter().GetResult();
+    }
+
+    [Params(100)]
+    public int PageSize { get; set; }
+    [Params(10)]
+    public int PageNumber { get; set; }
+
+
+    [Benchmark]
+    public async Task GetDocuments_Sequentially()
+    {
+        var storage = new DocumentStorage(_setup.DocumentsTableClient);
+        await storage.GetPageContentSequentiallyAsync(PageSize, PageNumber);
+    }
+
+    [Benchmark]
+    public async Task Get0Documents_RowKeysRange()
```

Comment:
```suggestion
    public async Task GetDocuments_RowKeysRange()
```

Assume this is a typo?